solidity/contracts/BondedECDSAKeepFactory.sol

```diff
@@ -444,7 +444,17 @@ contract BondedECDSAKeepFactory is
         return tokenStaking.hasMinimumStake(_operator, address(this));
     }
 
-    /// @dev Gets the stake balance of the specified operator.
+    /// @notice Checks if the factory has the authorization to operate on stake
+    /// represented by the provided operator.
+    ///
+    /// @param _operator operator's address
+    /// @return True if the factory has access to the staked token balance of
+    /// the provided operator and can slash that stake. False otherwise.
+    function hasAuthorization(address _operator) public view returns (bool) {
```

Comment:
We've got `isOperatorRegistered` and `isOperatorEligible` functions already, maybe we should name this function `isOperatorAuthorized` for consistency?